pages/events.jsx

```diff
@@ -120,7 +120,9 @@ var MakerPartyExample = React.createClass({
                 <span className="span-content label-tag">Participants</span><span className="span-content">{this.props.participants}</span>
               </div> : null
             }
-            <div className="description">{this.props.description}</div>
+            <div>
+              <p className="description" dangerouslySetInnerHTML={{__html: this.props.description}}></p>
+            </div>
```

Comment:
https://github.com/mozilla/teach.webmaker.org/blob/develop/pages/activities.jsx#L98 should probably be wrapped in `<p>` as well but we should probably wait till we decide on https://github.com/mozilla/teach.webmaker.org/issues/1083 before we make the change.